IBPSA/Fluid/Sources/PropertySource_h.mo

```diff
@@ -0,0 +1,62 @@
+within IBPSA.Fluid.Sources;
+model PropertySource_h
+  "Model for overriding fluid properties that flow through the component, using specific enthalpy input"
+  parameter Boolean use_h_in= false
+    "Set to true to get the specific enthalpy from the input connector"
+    annotation(Evaluate=true, Dialog(group="Inputs"));
+
+  extends IBPSA.Fluid.Sources.BaseClasses.PartialPropertySource;
```

Comment:
Move up

Reply:
This order is deliberate, otherwise the parameters are ordered in a non-logical sequence in the parameters window of dymola.